GitUI/Translation/English.xlf

```diff
@@ -133,10 +133,6 @@ This action will be performed without warning while checking out branch.</source
         <source>Appearance</source>
```

Comment:
Removed the duplicated strings. Essentially, these are being assigned to other controls in the "runtime" constructor; and hence not necessary.